common/lib/api-client/transformers/index.js

```diff
@@ -0,0 +1,9 @@
+const _transform = require('./_transform')
+const personEscortRecordTransformer = require('./person-escort-record.transformer')
+const personTransformer = require('./person.transformer')
+
+module.exports = {
+  _transform,
```

Comment:
As mentioned above, this naming is styled like a private variable, but it is being exported. I wonder if there's a better name for it instead.